test/uk/gov/hmrc/eeittadminfrontend/controllers/DeltaControllerSpec.scala

```diff
@@ -198,7 +203,7 @@ class DeltaControllerSpec
         .withSession("token" -> "someGoogleAuthenticationToken")
```

Comment:
There is a lot of duplication in this file

Reply:
purely for time efficiency as the repository is to be decommissioned soon and the urgency for the particular task